core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/dataset/DatasetController.java

```diff
@@ -116,9 +118,9 @@ public DatasetController(DatasetDtoModelAssembler datasetDtoModelAssembler,
 
     @GetMapping("/{uuid}/implementations")
     @PreAuthorize("hasPrivilege('DATASETS_READ')")
-    public DatasetDto get(@PathVariable UUID uuid) {
+    public Set<DatasetImplementationDto> get(@PathVariable UUID uuid) {
         return datasetService.get(new DatasetKey(uuid))
```

Comment:
Can you create a 'getDataImplementations(UUID datasetUuid) in the DatasetDtoRepository